Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -33,6 +34,11 @@ class GLContextWidget final : public GUI::Frame {
 public:
     bool load(const String& fname);
 
+    bool rotate_x = true;
+    bool rotate_y = false;
+    bool rotate_z = true;
+    float rotation_speed = 1.f;
```

Comment:
These should be private, have an `m_` prefix and appropriate getters and setters. Looks like `toggle_rotate_{x,y,z}()` and `set_rotation_speed()` are sufficient.